src/firmware/main/io/receive.c

```diff
@@ -33,8 +35,6 @@
 #include "io/motor.h"
 #include "io/mrf.h"
 #include "main.h"
```

Comment:
same as above

Reply:
Similar comment to above, out-of-scope for this PR.